examples/plotting/checkout_form.py

```diff
@@ -1,3 +1,11 @@
+'''This example shows a sample checkout form created using bokeh widgets.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.models.widgets.button, bokeh.models.widgets.checkbox, bokeh.models.widgets.groups, bokeh.models.widgets.inputs
+    :refs: :ref:`userguide_models` > :ref:`userguide_models_widgets`
```

Comment:
Sorry one more change, I missed this earlier. We stopped doing the manual "breadcrumbs" because it was too much of a pain to maintain. This just needs to be
```suggestion
    :refs: :ref:`userguide_models_widgets`
```

Reply:
Alright sir.